controllers/clustercurator_controller.go

```diff
@@ -67,5 +70,27 @@ func (r *ClusterCuratorReconciler) Reconcile(req ctrl.Request) (ctrl.Result, err
 func (r *ClusterCuratorReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&clustercuratorv1.ClusterCurator{}).
+		WithEventFilter(newClusterCuratorPredicate()).
 		Complete(r)
 }
+
+func newClusterCuratorPredicate() predicate.Predicate {
+	return predicate.Funcs{
+		UpdateFunc: func(e event.UpdateEvent) bool {
+			newClusterCurator, okNew := e.ObjectNew.(*clustercuratorv1.ClusterCurator)
+			oldClusterCurator, okOld := e.ObjectOld.(*clustercuratorv1.ClusterCurator)
+			if okNew && okOld {
+				if !reflect.DeepEqual(newClusterCurator.Status, oldClusterCurator.Status) {
```

Comment:
what if spec is updated & status also updated? guess it won't likely happen as status is subresource, but just curious 😂

Reply:
status updates are separate. so spec and status won't update together